### ISSUE TITLE
Keyboard input stops working

### ISSUE THREAD

**Issue by jformoe** [Microsoft/vscode#36930]
Since I installed this version of VSCode, sporadically I have found the keyboard input to stop working. It does not have to do with memory pressure or resource constraints. When it happens, pressing space or enter does nothing. When I switch to another open App (TextEdit, Safari) my keystrokes are accepted. I have found that closing and re-opening the file usually fixes it, if not, I close/restart VSCode.

- VSCode Version: Code 1.17.2 (b813d12980308015bcd2b3a2f6efa5c810c33ba5, 2017-10-16T13:47:41.889Z)
- OS Version: Darwin x64 15.6.0
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|1.4.0

---

Steps to Reproduce:

1.
2.

Reproduces without extensions: Yes/No

**Comment by dbaeumer** [comment]
@jformoe do you have any steps to reproduce this? When it happens does you see high CPU load. If not, could you do the following:

- Help > Toggle Developer Tools
- Source Tab
- try to break / pause execution to see where code is hanging

If you do have high CPU load could you provide us with the process details (command line) of the process that does have high CPU load.

**Comment by jformoe** [comment]
No, I have found that copy & paste also stops working, I agree it is a duplicate, thanks!
